src/java/com/phenix/pct/PCTProxygen.java

```diff
@@ -263,13 +254,6 @@ private void executeProxygen(File pxgFile) throws BuildException {
             } catch (IOException caught) {
                 cleanup();
                 throw new BuildException("Unable to parse log file", caught);
-            } finally {
-                if (reader != null) {
-                    try {
-                        reader.close();
-                    } catch (IOException uncaught) {
-                    }
-                }
             }
         } else {
             log("Unable to read log file : " + pxgLogFile.getAbsolutePath(), Project.MSG_WARN);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') NullPointerException might be thrown as 'pxgLogFile' is nullable here [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS2259)